nanopub/setup_profile.py

```diff
@@ -6,12 +6,11 @@
 from typing import Union, Tuple
 
 import click
-from rdflib import Graph, FOAF, BNode, Literal, URIRef
+import rdflib
 
-from nanopub import profile, Publication, NanopubClient
+from nanopub import profile, Publication, NanopubClient, namespaces
```

Comment:
I changed this to be more consistent with the rest of the library.

Reply:
Yep, I prefer this more explicit code